chipper/start.sh

```diff
@@ -58,13 +59,17 @@ elif [[ ${STT_SERVICE} == "whisper.cpp" ]]; then
     export LIBRARY_PATH="../whisper.cpp"
     ./chipper
   else
-    export C_INCLUDE_PATH="../whisper.cpp"
-    export LIBRARY_PATH="../whisper.cpp"
+    export C_INCLUDE_PATH=~/wire-pod/whisper.cpp
```

Comment:
wire-pod isn't necessarily located in this path

Reply:
There's a relative import above it, this line is redundant. I'm still modifying this file to fine tune it.